crates/runtime/src/embeddings/udtf.rs

```diff
@@ -464,9 +465,22 @@ impl TableProvider for VectorSearchUDTFProvider {
                 false,
             )],
             input: Arc::new(proj),
-            fetch: limit,
+            // i.e., the order limit passed down via the UDTF invocation
+            fetch: self.args.limit,
         });
 
-        state.create_physical_plan(&sort).await
+        /* TODO: for a query such as
+           `select count(*) from vector_search(realtime_reviews, 'disappoint', 10) limit 5;`
+            the resultant physical plan should have a GlobalLimitExec for the query limit,
+            but it gets lost??
+        */
+        let limit = LogicalPlan::Limit(Limit {
+            skip: None,
+            fetch: limit
+                .map(|l| Box::new(Expr::Literal(ScalarValue::UInt64(u64::try_from(l).ok())))),
```

Comment:
If you can't convert to a `u64`, is `fetch` with `Some(Box<Expr::Literal(ScalarValue::UInt64(None))>)` the same as `None`?